src/service/position-observer/position-observer-impl.js

```diff
@@ -152,12 +155,12 @@ export class PositionObserver {
       const entry = this.entries_[i];
 
       if (opt_force) {
```

Comment:
These two if statements belong in `Entry#update` as well.

Reply:
Ah! 👍